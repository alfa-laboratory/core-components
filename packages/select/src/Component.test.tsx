import React from 'react';
import { render } from '@testing-library/react';

import { Select } from './index';

describe('Snapshots tests', () => {
    it('should match snapshot', () => {
        expect(render(<Select options={[]} />)).toMatchSnapshot();
    });
});
