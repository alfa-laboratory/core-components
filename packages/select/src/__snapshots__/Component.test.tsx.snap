// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots tests should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        aria-expanded="false"
        aria-haspopup="listbox"
        aria-owns="downshift-1-menu"
        class="component"
        role="combobox"
        tabindex="-1"
      >
        <div
          class="component"
        >
          <div
            class="component s block hasRightAddons"
          >
            <div
              aria-controls="downshift-1-menu"
              aria-labelledby="downshift-1-label"
              class="field inner"
              id="downshift-0-input"
              tabindex="0"
            >
              <div
                class="inputWrapper"
              >
                <div
                  class="input"
                >
                  <div
                    class="contentWrapper"
                  />
                </div>
              </div>
              <div
                class="addons rightAddons"
              >
                <span
                  class="arrow arrow"
                />
              </div>
            </div>
            
          </div>
        </div>
        <div
          aria-labelledby="downshift-1-label"
          class="listWrapper"
          id="downshift-1-menu"
          role="listbox"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      aria-expanded="false"
      aria-haspopup="listbox"
      aria-owns="downshift-1-menu"
      class="component"
      role="combobox"
      tabindex="-1"
    >
      <div
        class="component"
      >
        <div
          class="component s block hasRightAddons"
        >
          <div
            aria-controls="downshift-1-menu"
            aria-labelledby="downshift-1-label"
            class="field inner"
            id="downshift-0-input"
            tabindex="0"
          >
            <div
              class="inputWrapper"
            >
              <div
                class="input"
              >
                <div
                  class="contentWrapper"
                />
              </div>
            </div>
            <div
              class="addons rightAddons"
            >
              <span
                class="arrow arrow"
              />
            </div>
          </div>
          
        </div>
      </div>
      <div
        aria-labelledby="downshift-1-label"
        class="listWrapper"
        id="downshift-1-menu"
        role="listbox"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
